tests/testthat/test-table_with_settings_ui.R

```diff
@@ -0,0 +1,92 @@
+library(shinytest2)
+
+testthat::test_that("maxi version", {
+  skip_on_cran()
+  skip_on_ci()
+  app <- AppDriver$new(
+    app_tws(),
+    name = "tws",
+    variant = "app_tws_max",
+    )
+  # check standard file naming
+  file_name <- app$get_value(input = "table_with_settings-downbutton-file_name")
+  testthat::expect_equal(
+    sub("_\\d{6}$", "", file_name),
+    paste0("table_", strftime(Sys.Date(), format = "%Y%m%d"))
+  )
+  app$expect_text("table")
+  app$set_window_size(width = 1392, height = 763)
```

Comment:
@gogonzo  the point is to launch one app and test on it instead of launching an app for every test to reduce run time. If we have no problems with run time, then it would look much nicer if we break the tests.

As far as i remeber, `skip_on_cran()` and `skip_on_ci()` can be only used in side `testthat`, can you confirm @BLAZEWIM ?

Reply:
Yes, we had a problem with `skip()`'s on CI side, so now we only use it inside the call. 
Comment on having mini and maxi versions: they represent two approaches: maxi is actually a recording of the app test, and mini is the minimum number of calls to have click and screenshot tested. It actually makes a difference, with setting lpp to `1` and then to `11` being one.

Reply:
So the breaking the `testthat` blocks into separate blocks for separate functionality is one of the things we had before but don't have now. We created a single app driver, then [ran multiple `testthat` blocks.](https://github.com/insightsengineering/teal.widgets/blob/26f392d0ef997d4da33c3c25b3a2157939494099/tests/testthat/apps/pws_base/tests/testthat/test-shinytest2.R).